dist/Leaflet.vector-markers.js

```diff
@@ -29,18 +31,21 @@
         if (options.icon) {
           icon = this._createInner();
         }
-        pin_path = L.VectorMarkers.MAP_PIN;
-        div.innerHTML = '<svg width="32px" height="52px" viewBox="0 0 32 52" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">' + '<path d="' + pin_path + '" fill="' + options.markerColor + '"></path>' + icon + '</svg>';
+        pin_path = options.map_pin || L.VectorMarkers.MAP_PIN;
+        div.innerHTML = '<svg width="' + options.iconSize[0] + 'px" height="' + options.iconSize[1] + 'px" viewBox="' + options.viewBox + '" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">' + '<path d="' + pin_path + '" fill="' + options.markerColor + '"></path>' + icon + '</svg>';
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Identifier 'pin_path' is not in camel case.